diff --git a/autogen.sh b/autogen.sh
index 35623fa..9907386 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -1,9 +1,9 @@
 #! /bin/sh
 
-if [ "$USER" = "root" ]; then
-	echo "*** You cannot do this as "$USER" please use a normal user account."
-	exit 1
-fi
+#if [ "$USER" = "root" ]; then
+#	echo "*** You cannot do this as "$USER" please use a normal user account."
+#	exit 1
+#fi
 if test ! -f configure.ac ; then
 	echo "*** Please invoke this script from directory containing configure.ac."
 	exit 1
